### ISSUE TITLE
[FR] 'array_merge(...)' misused: inversed order

### ISSUE THREAD

**Issue by orklah** [kalessil/phpinspectionsea#1224]
Description:
--
We already have 'array_merge(...)' misused inspection to deal with cases like these:
```php
$array = array_merge($array, array($elem));
```
It suggest replacing array_merge with $array[] = syntax.

But I was surprised it didn't trigger with
```php
$array = array_merge(array($elem), $array);
```

The previous fix would indeed change the behaviour slightly because the order of the array will be changed.

I had a couple of suggestions for this case:
1. Warn that the order may change with $array[] = syntax but suggest it anyway
2. Add a setting to allow this suggestion. That way, we won't suggest false-positive by default
3. Flag the line but don't suggest QF, only a label to check the doc and suggest replacements here.
4. suggest this QF:
```php
array_unshift($array, $elem);
``` 
(Note: array_unshift pass array by reference, we may want to avoid this QF if $array is a function call instead of a variable.)

Thanks!

**Comment by kalessil** [comment]
Look good, will implement shortly!

**Comment by kalessil** [comment]
Implemented!
